sdk/include/reactos/shlwapi_undoc.h

```diff
@@ -122,6 +122,8 @@ HRESULT WINAPI SHPropertyBag_WritePOINTL(IPropertyBag *ppb, LPCWSTR pszPropName,
 HRESULT WINAPI SHPropertyBag_WritePOINTS(IPropertyBag *ppb, LPCWSTR pszPropName, const POINTS *ppts);
 HRESULT WINAPI SHPropertyBag_WriteRECTL(IPropertyBag *ppb, LPCWSTR pszPropName, const RECTL *prcl);
 
+HRESULT WINAPI SHCreatePropertyBagOnMemory(DWORD dwMode, REFIID riid, IPropertyBag **pppb);
```

Comment:
Maybe adding some documentation link like https://www.geoffchappell.com/studies/windows/shell/shlwapi/api/propbag/getviewstate.htm
or something (also it seems it's exported in vista+, in which case having a _WIN32_VERSION guard would be needed)

Reply:
`SHCreatePropertyBagOnMemory` is also exported in Win2k3 as ordinal function.

Reply:
Added comments.

Reply:
Documentation link is not found.

Reply:
Oh yes correct, it was one of the others.
This one should work: http://undoc.airesoft.co.uk/shlwapi.dll/SHCreatePropertyBagOnMemory.php
In addition, https://www.geoffchappell.com/studies/windows/shell/shlwapi/api/propbag/index.htm has links for the other functions.

Reply:
http://undoc.airesoft.co.uk/shlwapi.dll/SHCreatePropertyBagOnMemory.php is added.

Reply:
Should I add `SHPropertyBag_...` links?

Reply:
If you personally find it useful, why not (I'm not forcing anything here as I don't have any preference). But if you want to add the links for all the other SHPropertyBag_ functions, just use the probag/index.htm one, no need I think to add each of the sublinks for each function.

Reply:
https://github.com/reactos/reactos/pull/5515

Reply:
Done.